### ISSUE TITLE
Omgwtfnzbs is not working as an Indexer

### ISSUE THREAD

**Issue by antonichoudhuri** [Radarr/Radarr#2591]
I get "No results were returned from your indexer, please check your settings." from clicking Test on the Add/Edit new Indexer modal.

Steps to repro:
1. Browse to Indexers tab (http://localhost:7878/settings/indexers)
2. Add new indexer, click Omgwtfnzbs, Presets --> Omgwtfnzbs
3. Enter in all info:
Name: Omgwtfnzbs
Enable RSS Sync: Yes
Enable Search: Yes
Username: ...
API Key: ...
Delay 30
10. Click Test

I turned on Verbose logging and still get the same Warning: "Invalid request Validation failed: -- No results were returned from your indexer, please check your settings."

I am sure my Username/API key are correct and valid as I can browse directly to the RSS link/URL using those parameters/values in the URL query string and get correct results.

Radarr Version: 0.2.0.980
OS: Windows 10 Pro

Debug Logs:
NzbDroneErrorPipeline | Invalid request Validation failed:   -- No results were returned from your indexer, please check your settings.

**Comment by fryfrog** [comment]
Don't use the built in pre-configured one, create a new newznab one and use their newznab api. They had (?have?) their own custom api from long ago, but using newznab is best.

**Comment by antonichoudhuri** [comment]
@fryfrog yep! that works. Thanks.

Still seems like it is an active bug. I propose fixing or removing that pre-configured one if it doesn't work. I will leave the issue here and let someone triage it.

**Comment by Qstick** [comment]
@antonichoudhuri are you VIP on OMG? The legacy API will return `this feature is for VIPs only - please upgrade to VIP` which Radarr interprets as no results.

**Comment by antonichoudhuri** [comment]
@Qstick I was able to get it to work without VIP.

I started debugging the issue and believe I was being throttled by OMG and Radarr was not handling the response gracefully. Given the rates which free users get (10 requests per day) this makes sense.

I suggest:
1. Make Radarr aware/handle the different responses of the OMG API. This might be hard as documentation is limited/sparse. Can at least start with (throttled and 404)
2. In the case of an error have Radarr save and then display the response from the web server or at least the http status code (worst case is 404) in the debug log. This will help easily diagnosis future issues.
